R/zchunk_batch_modeltime.xml.R

```diff
@@ -0,0 +1,36 @@
+#' module_modeltime_batch_modeltime.xml
+#'
+#' Construct XML data structure for \code{modeltime.xml}.
```

Comment:
Yes, that is correct.  I know it might not be intuitive at first though that these should be their chunks and not have the Level 2 chunk just generate the XML.  After all in the old data system it _appears_ that is how it was done.

However I ultimately decided to design it like this for a couple of reasons:
1. There are a handful of cases where data is added to an XML file from multiple Level 2 .R files.
2. Fairly commonly the data output of the Level 2 files are in fact used again in some other Level 2 .R file in addition to in the XML file creation.  So we want to make sure we declare those as outputs of Level 2 processing as well instead of just .xml files.

That being said I could be convinced to just collapse some of the simple cases into just a single chunk.